src/psyclone/tests/psyir/symbols/symboltable_test.py

```diff
@@ -603,8 +603,8 @@ def test_swap_symbol_properties():
     # symbol name is used as the key in the symbol table).
     with pytest.raises(ValueError) as excinfo:
```

Comment:
Since we're here, this file should really be named `symbol_table_test.py`. Would you mind moving it?

Reply:
Done. Also psyir/symbols/symboltable.py to psyir/symbols/symbol_table.py

Reply:
Thank you :-)